packages/genesys-spark-components/src/components/beta/gux-day/gux-day.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  JSX,
+  Prop,
+  State,
+  Watch
+} from '@stencil/core';
+import { getDesiredLocale } from 'i18n';
+import * as sparkIntl from '../../../genesys-spark-utils/intl';
+
+@Component({
+  styleUrl: 'gux-day.scss',
+  tag: 'gux-day',
+  shadow: { delegatesFocus: true }
+})
+export class GuxDay {
+  @Element()
+  root: HTMLElement;
+
+  /* The date of the day in ISO format, e.g. 2024-10-22 */
+  @Prop()
+  day: string;
+
+  /* JS Date object, derived from the `day` attribute */
+  @State()
+  date: Date;
+
+  /* Watcher to sync the internal date with the day attribute on changes */
+  @Watch('day')
+  onDateAttrChange(): void {
+    this.date = new Date(this.day);
```

Comment:
@MattCheely welcome to developing with Javascript Dates in the western hemisphere. This does not work as expected for timezones behind GMT.
![Screenshot 2024-12-03 at 11 28 48](https://github.com/user-attachments/assets/7fbfe8a1-ab92-4b3c-8e47-c42f111b89cc)
The calendar for someone in the US for example will be shifted a day and be incorrect